src/_guides/libraries/futures-error-handling.md

```diff
@@ -1,19 +1,24 @@
 ---
-layout: article
+layout: default
 title: "Futures and Error Handling"
 description: "Everything you wanted to know about handling errors and exceptions when working with Futures (but were afraid to ask)."
-written: 2013-03-07
-updated: 2013-05-30
-category: libraries
 ---
 
-_Written by Shailen Tuli <br>
-March 2013 (updated May 2013)_
-
-This article covers the subject of error handling when dealing with Futures.
-If you are unfamiliar with the general concepts behind Futures, we
-recommend you first read
-[Asynchronous Programming: Futures](/tutorials/language/futures).
+In many cases, when you want to write asynchronous code in Dart,
```

Comment:
I was confused by this intro. _Futures and Error Handling_ doesn't seem (on the surface) to preclude using async-await & try-catch.

Perhaps we need to add a little section on async-await & try-catch, so this page can be an all-in-one resource.

If not, than this page needs to be renamed and its scope better defined.